styleguide/source/_patterns/03-organisms/wayfinder.twig

```diff
@@ -0,0 +1,22 @@
+<section class="ama_wayfinder">
+  <div class="ama_wayfinder--grid container">
```

Comment:
The "grid" is a sub component of wayfinder, not a modification of it. Therefore, it should be using a double underscore to separate it out (e.g. `ama__wayfinder__grid`). Also it should be a double underscore between `ama` and the block level element, it's a bit hard to tell here if it's one or two there.